cypress/support/index.js

```diff
@@ -1,23 +1,25 @@
 // ***********************************************************
-// This example support/index.js is processed and
-// loaded automatically before your test files.
-//
-// This is a great place to put global configuration and
-// behavior that modifies Cypress.
-//
-// You can change the location of this file or turn off
-// automatically serving support files with the
-// 'supportFile' configuration option.
-//
-// You can read more here:
-// https://on.cypress.io/configuration
+// Main support file for test. It's processed and loaded automatically before all test files.
+// You can read more here: https://on.cypress.io/configuration
 // ***********************************************************
 
 // Import commands.js using ES2015 syntax:
-import './commands'
+import "./commands";
 
 // Alternatively you can use CommonJS syntax:
 // require('./commands')
+before(function() {
+    cy.ganacheTakeSnapshot();
+    cy.issueTo(100000); // to make tests independent. issue to accounts[0] by default (amount with token decimals)
```

Comment:
tell me more! :)

Reply:
are you referring to the comment leftover from the cypress example ? (removed)

Reply:
I don't understand what's happening here with both `.ganacheTakeSnapshot();` and `.issueTo(100000);` calls.